monitoring/prometheus.go

```diff
@@ -0,0 +1,125 @@
+package monitoring
+
+import (
+	"net/http"
+	"sync"
+
+	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
```

Comment:
So this repo is now ready only: https://github.com/grpc-ecosystem/go-grpc-prometheus#deprecated-go-grpc-interceptors-for-prometheus-monitoring

We should update import paths to keep with the maintained version.

Reply:
Updated to `github.com/grpc-ecosystem/go-grpc-middleware/providers/prometheus` :+1: